wrappers/internal/service_unit_gen_test.go

```diff
@@ -487,6 +556,84 @@ apps:
 	c.Check(string(generatedWrapper), Equals, expectedUserAppService)
 }
 
+func (s *serviceUnitGenSuite) TestGenerateSnapGraphicalUserServiceFile(c *C) {
+	yamlText := `
+name: snap
+version: 1.0
+apps:
+    app:
+        command: bin/start
+        stop-command: bin/stop
+        reload-command: bin/reload
+        post-stop-command: bin/stop --post
+        stop-timeout: 10s
+        daemon: simple
+        daemon-scope: graphical-user
+`
```

Comment:
Why is graphical user and not graphical session?

I'd follow systemd here. Or we create an inconsistency that may create confusion.

I acknowledge this is for the user services only, but there are no such things as system graphical sessions.

If we really want to keep the user in, I'd use `user-graphical-session`.

This is relevant for the multi-seat cases, since if an user has more sessions, the service is still bound to the current graphical session